pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/define-infrastructure/index.md

```diff
@@ -101,7 +103,23 @@ Konvoy needs to know how to access your cluster hosts. This is done using invent
 1.  Apply the new infrastructure file with the command:
 
     ```shell
-    kubectl apply -f preprovisioned_inventory.yaml
+    envsubst < preprovisioned_inventory.yaml | kubectl apply -f -
     ```
 
+## Prepare your machines
+
+In order for DKP to fully install, it is recommended you stop `firewalld` if enabled so DKP can fully deploy.
+
+You will need to check to see if `firewalld` is running. SSH into all the machines you are deploying DKP onto, and check to see if it is running:
+
+```sh
+systemctl status firewalld
+```
+
+If it is active, you will need to stop `firewalld`:
+
+```sh
+systemctl stop firewalld
+```
+
```

Comment:
We might also want to remove any preconfigured ip table filter rules as well

Reply:
@faiq good call - I'll iterate.